### DISCUSSION TITLE
Error using with winter OS

### DISCUSSION THREAD

**Discussion by seb090819** [farag2/Sophia-Script-for-Windows#501] [Q&A] [answered]
Hello, I wanted to know if you have any idea how I can use the script in the modified version of Windows "Winter OS Rev09"
![imagen_2023-06-11_200702683](https://github.com/farag2/Sophia-Script-for-Windows/assets/136283455/4606e1a9-d5a3-436f-8ad6-03f331792921)

**Comment by farag2** [comment, marked as answer]
Hi. technically you can bypass the check but I strongly recommend you to install genuine Windows 10 and avoid in the future installing any home brew images like this one.
Why WinterOS was blocked: it provides Edge removing and something else. I do not remember what actually else. Anyway, "normal" custom made image won't provide such destructive feature. That's why it's blocked. Also, what is the purpose of using? You want use a speed up image? This is impossible. You want a custom image? Just apply Sophia Script on any original image. What's the purpose? Or you believed that blogger that it runs smoother, send less telemetry data and weighs less than it should? lol **he's just a scammer**.

https://github.com/farag2/Sophia-Script-for-Windows/blob/aa233d94607c193477c865476c54a586080c328b/src/Sophia_Script_for_Windows_11/Module/Sophia.psm1#L402

https://www.youtube.com/watch?v=L0cj_I6OF2o
